### PR TITLE
Cosmetic README updates

### PR DESCRIPTION
**Pull request by rbren**
**Short description of the problem this fixes or functionality that this introduces. This may be used for the CHANGELOG**

Cleaning up the readme a bit after several months of iteration :)

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**
* Clean up badges
* Update logo
* Move some docs from the README into getting-started.md

See changes here: https://github.com/All-Hands-AI/OpenHands/blob/4f742e3d3ae52bf65f87c8b4d73e9db753ad61fd/README.md

---
**Link of any specific issues this addresses**

### CODE REVIEW

File: docs/modules/usage/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+# ⚡Getting Started
+
+## System Requirements
+* Docker version 26.0.0+ or Docker Desktop 4.31.0+
+* You must be using Linux or Mac OS
+  * If you are on Windows, you must use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+## Installation
+The eaisest way to run OpenHands is in Docker. Use `WORKSPACE_BASE` below to
```
Comment (mamoodi) [inline docs/modules/usage/getting-started.md:13]: easiest

File: docs/modules/usage/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 3
```
Comment (mamoodi) [inline docs/modules/usage/getting-started.md:2]: Make this 2 now that you changed the other one

File: docs/modules/usage/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+* Docker version 26.0.0+ or Docker Desktop 4.31.0+
+* You must be using Linux or Mac OS
+  * If you are on Windows, you must use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+## Installation
+The eaisest way to run OpenHands is in Docker. Use `WORKSPACE_BASE` below to
+specify which folder the OpenHands agent should modify.
+
+See the [Getting Started](https://docs.all-hands.dev/modules/usage/getting-started) guide for
+system requirements and more information.
```
Comment (mamoodi) [inline docs/modules/usage/getting-started.md:17]: You are already in the Getting Started guide here

File: docs/modules/usage/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    -e SANDBOX_RUNTIME_CONTAINER_IMAGE=ghcr.io/all-hands-ai/runtime:0.9.2-nikolaik \
+    -e SANDBOX_USER_ID=$(id -u) \
+    -e WORKSPACE_MOUNT_PATH=$WORKSPACE_BASE \
+    -v $WORKSPACE_BASE:/opt/workspace_base \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -p 3000:3000 \
+    --add-host host.docker.internal:host-gateway \
+    --name openhands-app-$(date +%Y%m%d%H%M%S) \
+    ghcr.io/all-hands-ai/openhands:0.9
+```
```
Comment (mamoodi) [inline docs/modules/usage/getting-started.md:32]: I guess we went back to duplicating the docker command? :D 
It's fine. Just got to make sure both get updated.

Comment (rbren) [reply docs/modules/usage/getting-started.md:32]: Yup :(

**Review by mamoodi — APPROVED** [review]

**Pull request merged**